shared/test/test_redis_table.rb

```diff
@@ -45,59 +44,58 @@ def test_redis_tables
                  @pubsub.publish_history
 
     value2 = {'foo' => 52}
-    table.insert(value2)
-    assert_equal [row(value, 1), row(value2, 2)], table.to_a
-    assert_equal row(value2, 2), table.fetch(2)
+    row2 = table.insert(value2)
+    assert_equal [row1, row2], table.to_a
+    assert_equal row2, table.fetch(2)
     assert_equal make_pubsub_event('shard1', 'table', {:action => 'insert', :id => 2}),
                  @pubsub.publish_history[2]
 
     value2a = {'foo' => 53}
-    updated = table.update(2, value2a)
-    assert_equal(row(value2a, 2), updated)
+    updated_row2 = table.update(2, value2a)
+    assert_equal updated_row2, table.fetch(2)
+    assert_equal row2['uuid'], updated_row2['uuid']
```

Comment:
The update also overrules any attempt to update a field with a new `uuid`; I'd like to see a test for that, too